.circleci/config.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       - checkout
       - run:
           command: |
-            sudo apt-get install yamllint python3-pkg-resources
+            sudo chmod +x ./.circleci/install-yamllint.sh && ./.circleci/install-yamllint.sh
```

Comment:
you can also change file mode locally and commit file mode changes to avoid problems with permission on CI.
Try to do something like `git update-index --add --chmod=+x .circleci/install-yamllint.sh`

Reply:
Thanks, fixed